### ISSUE TITLE
query_scope is ignored for Tempo in single-cluster environment

### ISSUE THREAD

**Issue by nvkv** [kiali/kiali#7658]
### Describe the bug

If you specify `query_scope` for configuration using Tempo provider over HTTP it will be completely ignored, unless your Tempo is running in multitennant mode (i. e. your traces have `cluster` tag).

### Expected Behavior

Generated TraceQL should include pairs specified in `query_scope`.

### What are the steps to reproduce this bug?

Tracing configuration:

```yaml
...
      tracing:
        enabled: true
        provider: "tempo"
        query_scope:
          whatever_tag: "magic_value"
        use_grpc: false
        namespace_selector: true
        in_cluster_url: ...
        health_check_url: ...
...
```

1. Run Kiali with `log_level: debug` and (adjusted) specified configuration
2. Try to fetch traces using Kiali 
3. Observe prepared TraceQL in logs, there is no `whatever_tag+=+"magic_value"` part

Proposed patch:

```patch
diff --git a/tracing/tempo/http_client.go b/tracing/tempo/http_client.go
index b6896355f..a5a1568e8 100644
--- a/tracing/tempo/http_client.go
+++ b/tracing/tempo/http_client.go
@@ -249,7 +249,7 @@ func (oc OtelHTTPClient) prepareTraceQL(u *url.URL, tracingServiceName string, q
 
        if len(query.Tags) > 0 {
                for k, v := range query.Tags {
-                       if k != models.IstioClusterTag && oc.ClusterTag {
+                       if k != models.IstioClusterTag {
                                tag := TraceQL{operator1: "." + k, operand: EQUAL, operator2: v}
                                queryPart = TraceQL{operator1: queryPart, operand: AND, operator2: tag}
```

### Environment
*Learn about how to determine versions [here](https://kiali.io/docs/faq/general/#how-do-i-determine-what-version-i-am-running).*

- **Kiali version:** 1.86
- **Istio version:**  1.22.3
- **Kubernetes impl:** Kubernetes
- **Kubernetes version:** v1.26.1
- **Other notable environmental factors:** N/A

**Comment by josunect** [comment]
> cc @jshaughn he was involved in that `query_scope` feature when it was first introduced.

I think this is just related to the Tempo implementation and it is related to https://github.com/kiali/kiali/pull/6832, I will verify if this change is working with multi cluster.

**Comment by jshaughn** [comment]
@josunect If you think this should be backported to v1.89 please add the backport-needed label and create the backport.  Otherwise the fix will wait for 2.0.

**Comment by josunect** [comment]
Added backport for 1.89
